### PR TITLE
#167484575 configured authors haven to use postgresql as db engine and sequelize

### PR DESCRIPTION
**Pull request by placideirandora**
#### What does this PR do?
It configures **Authors Haven** to use **PostgreSQL** as a database engine and **Sequelize** as **ORM** (Object Relational Mapping).

#### Description of Task to be done?
1. Setting up required dependencies/packages.
2. Configuring Sequelize to use the PostgreSQL database.
3. Testing the database connection by creating the server with Express.

#### How should this be manually tested?
1. Clone this branch.
2. Do **npm install**.
3. Create two databases called: **authors-haven** for development and **authors-haven-test** for testing.
4. Finally, Do **npm start** to check if you get a console message telling you that: **Database Connected!**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,17 +4,22 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "nodemon --exec babel-node ./src/index.js"
```
Comment (mkiterian) [inline package.json]: nodemon should only be used in dev environments.  The start script should simply run the app with `node` add a new `dev` script which uses `nodemon`

File: package.json [hunk 2]
```diff
@@ -4,17 +4,22 @@
...
   },
   "author": "Andela Simulations Programme",
   "license": "MIT",
   "dependencies": {
-    "body-parser": "^1.18.3",
+    "@babel/cli": "^7.5.5",
```
Comment (mkiterian) [inline package.json]: these babel packages should be under dev dependencies

**Review by mkiterian — CHANGES_REQUESTED** [review]
It'll also be helpful to add a .env.sample file with fake keys so that other devs know which environment variables need to be set

File: package.json (hunk 2 above)
Comment (raymond42) [reply package.json]: Babel packages should  be in dev dependencies as @mkiterian have said

**Review by raymond42 — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -4,17 +4,19 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "babel-node ./src/index.js",
```
Comment (mkiterian) [inline package.json]: remove the start script for now

**Review by mkiterian — APPROVED** [review]
Looks good

**Pull request merged**